src/var/clearInterval.js

```diff
@@ -0,0 +1,3 @@
+define(function() {
+	return window.clearInterval;
```

Comment:
> Most of these changes landed in 76df9e4e389d80bff410a9e5f08b848de1d21a2f. AFAIK @gibson042 advocated creating variables for every one of them so maybe he'll want to say sth here.

[Specifically](https://github.com/jquery/jquery/pull/1949#discussion_r22042857), "Since we're defining `docElem` here and in offset.js, let's make it a variable too". IIRC, it was just a redundancy concern as @markelog suggests:

> So every host object should be modulized? Or just those that are used more than in one module?

(we actually _still_ have a similar situation in the form of `div = document.createElement( "div" )`: [attributes/support](https://github.com/jquery/jquery/blob/63f19a95b9bfffdc7b8800f89a9def768cc3aebd/src/attributes/support.js), [css/support](https://github.com/jquery/jquery/blob/63f19a95b9bfffdc7b8800f89a9def768cc3aebd/src/css/support.js), [data/support](https://github.com/jquery/jquery/blob/63f19a95b9bfffdc7b8800f89a9def768cc3aebd/src/data/support.js), [event/support](https://github.com/jquery/jquery/blob/63f19a95b9bfffdc7b8800f89a9def768cc3aebd/src/event/support.js), [manipulation/support](https://github.com/jquery/jquery/blob/63f19a95b9bfffdc7b8800f89a9def768cc3aebd/src/manipulation/support.js))

Provided naked use of the globals will fail linting, though, I personally don't care. Come to think of it, I suppose it would be fine to drop _all_ the modules and prefix everything with `window.`—whatever's smaller.